tests/test_general.py

```diff
@@ -685,50 +686,41 @@ def test_general_index_access():
         h[0:10:20j, 0:5:10j, "hello", False, 5]
 
 
-def test_general_density():
+def test_general_proxy():
     """
-        Test general density -- whether Hist can be accessed by index.
+        Test general proxy -- whether Hist proxy works properly.
     """
-
-    for data in range(10, 20, 10):
-        h = Hist(axis.Regular(10, -3, 3, name="x")).fill(np.random.randn(data))
-        assert round(sum(h.density()), 2) == round(10 / 6, 2)
-
-
-# henry's tests
-def test_histogram_quick_construction():
-    h = Hist.Regular(10, 0, 1, name="x")
-    h.fill([0.5, 0.5])
+    h = Hist.Regular(10, 0, 1, name="x").fill([0.5, 0.5])
     assert h[0.5j] == 2
 
+    h = (
+        Hist()
+        .Regular(10, 0, 1, name="x")
+        .Regular(10, -1, 1, name="y")
+        .fill([0.5, 0.5], [-0.2, 0.6])
+    )
 
-def test_histogram_quick_constrution():
-    h = Hist.Regular(10, 0, 1, name="x").Regular(10, -1, 1, name="y")
-    h.fill([0.5, 0.5], [-0.2, 0.6])
-    assert h[0.5j, -0.2j] == 1
-
-
-def test_histogram_unnamed_axes():
-    Hist(axis.Regular(10, 0, 1), axis.Regular(20, -3, 3))
-
-
-def test_histogram_loc():
-    h = Hist(axis.Regular(100, -50, 50))
+    assert h[0.5j, -0.2j] == h[bh.loc(0.5), bh.loc(0.6)] == 1
 
-    h.fill([0, 2.1])
-    h[0j] == 1
-    h[2.1j] == 1
+    # add axes to existing histogram
+    with pytest.raises(Exception):
+        Hist().Regular(10, 0, 1, name="x").fill([0.5, 0.5]).Regular(10, -1, 1, name="y")
 
+    with pytest.raises(Exception):
+        Hist(axis.Regular(10, 0, 1, name="x")).Regular(10, -1, 1, name="y")
 
-def test_histogram_rebin():
```

Comment:
You added this one in a previous PR, I think, in an indexing test?

Reply:
Yep!